### ISSUE TITLE
Process Explorer: NVDA does not announce CPU% and Memory(MB)

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/vscode#52578]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
ScreenReader Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Go to Help--> Open Process Explorer

**Actual:**
NVDA does not announce any information about the data changing dynamically in CPU% and Memory(MB). 

**Expected:**
NVDA should announce some information about the data which is changing dynamically in Column1(CPU%) and Column2(Memory)

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know about the data which are changing.

Attachment for Reference: 
[A11y VSCode Process Explorer NVDA does not read content changing.pptx](https://github.com/Microsoft/vscode/files/2123068/A11y.VSCode.Process.Explorer.NVDA.does.not.read.content.changing.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@RMacfarlane is it ok if we look into this in September? It seems low hanging and we need this to achieve a better accessibility grade.

**Comment by RMacfarlane** [comment]
This is actually more difficult to fix than I thought it would be - the whole table's HTML gets updated every second, only the top level element is in the HTML that gets initially loaded. Putting `aria-live: "polite"` on that element causes NVDA to start reading the entire table when values have changed, instead of just the specific entry. And putting `aria-live` on any specific cell doesn't work since the cell needs to be in the document when it first gets loaded for NVDA to listen for changes to it.

I opted for adding `aria-live: "polite"` on the top level so that at least something is read on update. Right now I can't think of a way to scope it down any further

**Comment by chrmarti** [comment]
It now reads a text every second that is too long to be read/understood within a single second. I'm not a screen reader user myself, so I can't tell for sure, but does that really fix it?

**Comment by sbatten** [comment]
Agree with @chrmarti 's observation. While something is now read, I think the bug itself should be re-opened and fixed later.

**Comment by isidorn** [comment]
Even though I agree that the solution is not optimal. There is no better way to fix this via aria spec.
Optimally we would read out more rarely. However none of the users complained on this and the original issue is fixed. Thus closing
